test_scripts/AppServices/RPCPassThrough/001_SendLocation_Handled_Request.lua

```diff
@@ -63,24 +36,51 @@ local successResponse = {
   info = "Request was handled by app services"
 }
 
-local sendLocationRequest = { 
-  name = "SendLocation",
-  hmi_name = "Navigation.SendLocation",
+local rpcRequest = { 
   params = {
     longitudeDegrees = 50,
     latitudeDegrees = 50,
     locationName = "TestLocation" 
   },
-  hmi_params = params
+  hmi_params = {
+    longitudeDegrees = 50,
+    latitudeDegrees = 50,
+    locationName = "TestLocation" 
+  }
 }
 
-local sendLocationResponse = { 
-  name = "SendLocation",
-  hmi_name = "Navigation.SendLocation",
-  params = successResponse,
-  hmi_params = { code = 0 }    
+local rpcResponse = { 
+  params = successResponse    
 }
 
+--[[ Local functions ]]
+local function PTUfunc(tbl)
+  --Add permissions for app1
+  local pt_entry = common.getAppServiceProducerConfig(1)
+  pt_entry.app_services.NAVIGATION = { handled_rpcs = {{function_id = 39}} }
+  tbl.policy_table.app_policies[common.getConfigAppParams(1).fullAppID] = pt_entry
+  --Add permissions for app2
+  pt_entry = common.getAppDataForPTU(2)
+  pt_entry.groups = { "Base-4" , "SendLocation" }
+  tbl.policy_table.app_policies[common.getConfigAppParams(2).fullAppID] = pt_entry
+end
+
+local function RPCPassThruTest()
+  local providerMobileSession = common.getMobileSession(1)
+  local mobileSession = common.getMobileSession(2)
+  
+  local cid = mobileSession:SendRPC("SendLocation", rpcRequest.params)
```

Comment:
Minor enough detail that I'll still approve without, but including the RPC name in the rpcRequest struct can still be useful, instead of hardcoding it here.

Reply:
I assume this would also include adding the hmi_name `Navigation.SendLocation`?

Reply:
Not if it isn't used in that specific test probably